CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 ## 4.46.0 (Unreleased)
 
+FEATURES:
+* **New Data Source:** `google_firebase_android_app` ([#4955](https://github.com/hashicorp/terraform-provider-google-beta/pull/4955))
+* **New Resource:** `google_cloud_run_v2_job` ([#4937](https://github.com/hashicorp/terraform-provider-google-beta/pull/4937))
+* **New Resource:** `google_cloud_run_v2_service` ([#4942](https://github.com/hashicorp/terraform-provider-google-beta/pull/4942))
+* **New Resource:** `google_gke_backup_backup_plan` (beta) ([#4948](https://github.com/hashicorp/terraform-provider-google-beta/pull/4948))
+* **New Resource:** google_firebase_storage_bucket ([#4951](https://github.com/hashicorp/terraform-provider-google-beta/pull/4951))
+
+IMPROVEMENTS:
+* `network_services`: added `origin_override_action` and `origin_redirect` to `google_network_services_edge_cache_origin` ([#4936](https://github.com/hashicorp/terraform-provider-google-beta/pull/4936))
+* bigquerydatatransfer: recreate `google_bigquery_data_transfer_config` for Cloud Storage transfers when immutable params `data_path_template` and `destination_table_name_template` are changed ([#4929](https://github.com/hashicorp/terraform-provider-google-beta/pull/4929))
```

Comment:
Wording is strange here, not sure how to fix it

Reply:
Yeah, I'm not sure why the issue wasn't a bug fix in the first place.